config/config.js

```diff
@@ -39,6 +39,15 @@ const config = {
             },
         ],
     },
+    multiSignatureRegistration: {
+        min: 2,
+        participants: [
+            "album pony urban cheap small blade cannon silent run reveal luxury glad predict excess fire beauty hollow reward solar egg exclude leaf sight degree",
```

Comment:
it only needs a list of participant passphrases as the public key is derived from it in the builder.